### ISSUE TITLE
Divide policy costs to ISIC codes within "other industries" category proportionally to spending on utilities, not output

### ISSUE THREAD

**Issue by jrissman** [Energy-Innovation/eps-us#108]
From Robbie on 10/29/20:

One challenge I’m running into is the way in which we divvy up policy costs to the other industries sector in assigning them to them the ISIC codes that make up that sector.

For example, food, tobacco, and beverage is a big industry in Virginia, but as a source of emissions/energy use in Virginia I suspect it is quite small. Nationally, another example is paper in pulp, but in the opposite direction.

Currently, when we assign policy costs the other industries ISIC codes, we do so proportionally based on BAU output, but I’m wondering if we shouldn’t instead do this based on BAU amount spent on the utilities sector (ISIC 35T39) from each ISIC code. This isn’t perfect, but it does a better job capturing how energy/emissions policies might affect certain ISIC codes more than others, relating this to the amount they spend on utilities instead of what they produce. This would likely better represent how energy and climate policies might affect the specific other industries. This would fix an issue where a very low energy intensity industry might end up with a very high economic impact from a policy, even though in reality it uses far less energy than other emitting industries.

For example: 
![image](https://user-images.githubusercontent.com/45801289/97590587-026a5080-19bc-11eb-991b-53dada8ececd.png)

I believe you calculate this all internally in Vensim by multiplying the BAU output variable by the DLIM value from the utilities sector to each sector, then dividing by the total spend on the utilities sector to develop a fraction of other industries costs that should be assigned to each ISIC code.

What do you think?

**Comment by jrissman** [comment]
I think this is a great idea, but it's a little trickier than it might seem at first glance, because the best way to divide the policy cost/savings up varies by sector, and by cash flow type within the Industry sector.

In the Transportation sector, the policy costs/savings are based on which industries own/operate the most vehicles.  (The costs/savings are on buying vehicles, buying fuel for vehicles, OM for vehicles, and buying EV chargers.)  Since transportation fuel for trucks isn't sold by the utilities sector, I don't think the utilities sector spending is a good proxy for how much each industry uses vehicles.  I don't think we can use spending on products from the refined petroleum fuels industry either, because a lot of industries buy petroleum products for purposes other than powering vehicles, such as for use as a chemical feedstock, or to power industrial machinery.  So if we're looking for a proxy for the amount of vehicles owned/used, output is probably the best we've got, since it should be somewhat correlated with the total volume of input materials and products being moved around.

In the Electricity sector, the only time we allocate any costs among industries based on output is demand response service payments, and I think utility spending is superior to output for allocating these.

In the Buildings sector, the costs relate to buildings, and I agree utility spending is superior to Output for allocating these, since the changes that happen in the buildings sector relate to energy use (and even if the change isn't directly energy-related, like improving insulation, it is more likely to be undertaken by firms with higher utility spending, since they'd see greater energy cost savings).

In the Industry sector, we assign most costs and savings to specific ISIC codes using other input data tables (such as a table for who gets the revenue from process emissions abatement expenses), and we only use output-based allocation for certain specific things.  We still use it for allocating (to ISIC codes within the "other industries" category):

1. Revenue from carbon tax rebates due to industries engaging in CCS
2. Revenue from changes in demand for the industries' products
3. Spending on fuels
4. Spending on labor or machinery directly caused by efficiency, fuel switching, or process emissions policies (not material efficiency or cost-driven production changes)

For (1) and (3), I think utility spending is decent for the U.S., since the utilities include natural gas distribution systems, and industries mostly use NG and electricity.  But it's not perfect for the U.S., because some industries use coal and/or petroleum products.  And it would be a pretty poor methodology for China, where industries predominantly use coal.  We can probably handle this by using each ISIC code's share of spending on the sum of the products of the relevant energy-supplying industries.

For (2), I think output-based allocation is the closest fit.  These industries are losing/gaining money based on changes in demand for their products, which is intermediated by the IO tables, and has little to do with how much energy those industries themselves use to produce the products.

For (4), it can thankfully be handled like 1 and 3, because spending on process emissions policies is entirely done by specifically-broken-out industries, not industries in the "other industries" category.  So that leave us with the changes caused by energy efficiency or fuel switching policies, which is just like 1 and 3.

So my plan at this point is to:

- in the Transportation sector, leave output-based allocation in place
- In the Electricity sector, switch to utility spend allocation
- In the Buildings sector, switch to utility spend allocation
- In the Industry sector, leave output-based allocation in place for cash flow type (2)
- In the Industry sector, allocate cash flow types (1) (3) and (4) based on the share of spending on the sum of the: utilities, coal, and refined petroleum products sectors, rather than based on utility sector spending alone.

**Comment by robbieorvis** [comment]
Thanks, this mostly makes sense.

My only comment is that for industry, we calculate the changes in production based in part on how an industry’s costs change in response to a policy, which is in turn dictated upstream by how we assign costs within the industry sector to other industries. So one concern I would still have with sticking without based allocation is where we assign passthrough costs based on output, instead of based on a basket of energy consumption, that in turn causes that industry’s production to fall more than it should.

For example, let’s say we just have two industries in the other industries category: paper/pulp and food. Paper and pulp might use 75% of the energy but produce 25% of the output, whereas food produces 75% of the output by consumes 25% of the energy.

If we were to put on a CCS policy and allocate costs based on output, the food industry would get 75% of the costs while the paper and pulp industry only 25%, even though energy demand is the inverse of that.

Perhaps you meant this would be covered by option 4 and industry and I misinterpreted it; I just wanted to flag that we may be estimating changes in industrial production based on upstream decisions that I want to make sure we correctly capture.

________________________________
Robbie Orvis
Director of Energy Policy Design
Phone: 415-799-2171
98 Battery Street, Suite 202
San Francisco, CA  94111
www.energyinnovation.org<http://www.energyinnovation.org/>
[cid:image001.jpg@01D0D699.20A24470]
________________________________

Check out our new book, Designing Climate Solutions: A Policy Guide for Low-Carbon Energy<https://www.amazon.com/Designing-Climate-Solutions-Policy-Low-Carbon/dp/1610919564>
Available wherever books are sold

[Policy Design book cover]

**Comment by jrissman** [comment]
In your example, the costs of implementing the CCS policy would be covered under (4), not under (2).

An example of a cash flow change covered under (2) would be:

- A policy package increases consumers' cash flow
- With their additional money, consumers demand more (or more costly) food, such as increasing meat consumption in a developing country.  They only slightly increase their purchases of paper products.
- The food industry gets lots more revenue but the paper industry gets slightly more revenue.

We're trying to divide up the increase in revenue to the "other industries" category between food and paper.  In this case, I think BAU output level probably better approximates what people (or other entities) would buy more of, rather than BAU spending on energy.

If this variable were only used for changes in consumer spending, we could use the IO tables to find what ISIC codes consumers actually buy stuff from, and allocate it that way.  But this variable is also used to allocate changes within "other industries" category due to changes in purchases by industry, or by government, or by foreign entities (exports), so we need a simple catch-all that encompasses what is usually being bought from the "other industries" category.  I see nothing better than "output" to represent this right now.

After doing #89, the "other industries" category will be much smaller, or maybe gone entirely.  If we break everything out by ISIC code and eliminate the "other industries" category, which I'm leaning toward doing, we will have no need to allocate cash flows within the "other industries" category anymore.  So I would consider whatever we decide to do here a temporary measure until #89 provides top precision.  (In fact, this whole GitHub issue, #108, may be largely obviated by completion of #89.  #108 is a temporary fix meant to tide us over until I do #89 sometime next year.)

**Comment by robbieorvis** [comment]
Ah okay.

Yes, that makes sense.

And I totally agree with you on this as a short term fix. I was hoping we could implement something like this to help address the few issues that have come up in testing the state models while we plan for the larger overhaul for sometime in 2021.

Thank you!

________________________________
Robbie Orvis
Director of Energy Policy Design
Phone: 415-799-2171
98 Battery Street, Suite 202
San Francisco, CA  94111
www.energyinnovation.org<http://www.energyinnovation.org/>
[cid:image001.jpg@01D0D699.20A24470]
________________________________

Check out our new book, Designing Climate Solutions: A Policy Guide for Low-Carbon Energy<https://www.amazon.com/Designing-Climate-Solutions-Policy-Low-Carbon/dp/1610919564>
Available wherever books are sold

[Policy Design book cover]

**Comment by jrissman** [comment]
Completed in 241ce11.  All the cash flows being divided up among ISIC codes within the "Other Industries" category should be more accurate now.  Closing this issue.

Oddly, although we took in lots of derived metrics from the input-output tables in 3.0.0, we never took a perfectly standard input-output table into Vensim.  I needed to do that as part of 241ce11, hence the new variable `io-model/SIOM`.  It relies on the same OECD data source we use elsewhere, plus the same BEA data to split up chemicals and pharmaceuticals, so it should look very familiar and be easy to adapt to other geographies in the same way you have adapted existing IO variables like `io-model/BObIC`, etc.
